$NetBSD: patch-modules_gui_qt4_Modules.am,v 1.1 2012/06/15 08:51:25 wiz Exp $

Workaround for:
/usr/pkg/share/automake-1.12/am/depend2.am: error: am__fastdepOBJCXX does not appear in AM_CONDITIONAL
/usr/pkg/share/automake-1.12/am/depend2.am:   The usual way to define 'am__fastdepOBJCXX' is to add 'AC_PROG_OBJCXX'
/usr/pkg/share/automake-1.12/am/depend2.am:   to 'configure.ac' and run 'aclocal' and 'autoconf' again
modules/gui/qt4/Makefile.am: error: Objective C++ source seen but 'OBJCXX' is undefined
modules/gui/qt4/Makefile.am:   The usual way to define 'OBJCXX' is to add 'AC_PROG_OBJCXX'
modules/gui/qt4/Makefile.am:   to 'configure.ac' and run 'autoconf' again.

--- modules/gui/qt4/Modules.am.orig	2011-12-08 18:00:26.000000000 +0000
+++ modules/gui/qt4/Modules.am
@@ -317,9 +317,9 @@ SOURCES_qt4 = 	qt4.cpp \
 		util/buttons/DeckButtonsLayout.cpp \
 		util/buttons/RoundButton.cpp
 
-if HAVE_DARWIN
-SOURCES_qt4 +=  util/searchlineedit_mac.mm
-endif
+#if HAVE_DARWIN
+#SOURCES_qt4 +=  util/searchlineedit_mac.mm
+#endif
 
 if HAVE_WIN32
 SOURCES_qt4 += main_interface_win32.cpp
